package.json

```diff
@@ -3,7 +3,6 @@
   "version": "1.0.0-rtm.4",
   "description": "A keyboard driver for cycle.js",
   "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
   "scripts": {
```

Comment:
Honestly I don't know what typings is for, and why we needed to begin with :D... And it's deprecated since typescript 2.0.

Anyway type definitions (*.d.ts) are generated correctly. So it seems superfluous to me.